styles/main.css

```diff
@@ -53,6 +60,14 @@ body {
     background-color: #D9D9D9;
 }
 
+.tips__tip {
+    width: 300px;
+    height: 350px;
+    padding: 0px 15px 10px 15px;
+    margin-bottom: 25px;
+    background-color: #D9C3C3;
+}
+
 .fish__image,
 .location__image {
     width: 330px;
```

Comment:
there's a couple places like this where having percentages could be more helpful than px lengths! makes it more responsive when the window gets resized